.env.prod.db-sample

```diff
@@ -1,3 +1,3 @@
-POSTGRES_USER=hello_django
-POSTGRES_PASSWORD=hello_django
-POSTGRES_DB=hello_django_prod
+POSTGRES_USER=hello_zm
```

Comment:
Proposition - use one .env file, we'll avoid mstakes and typos/

Reply:
ok